src/clj/nnts2/db.clj

```diff
@@ -15,3 +15,11 @@
 (defn rollback
   []
   (repl/rollback (migration-config)))
+
+(defn lein-migrate-db [& args]
+  (config/read (keyword (first args)))
+  (migrate))
+
+(defn lein-rollback-db [& args]
+  (config/read (keyword (first args)))
+  (rollback))
```

Comment:
It would be good if we can rollback a single migration via command line. As you have already done this much, if you give option to read extra argument, rollbacks can be made more useful

Reply:
This does exactly that - it rolls back by 1 migration id.